### PR TITLE
Asap7 drc lvs

### PR DESCRIPTION
**Pull request by yathAg**
Implements DRC check for final GDSII in ASAP7. The check accurately compares the rules defined in the ASAP7 pdk design reference manual with the final GDSII using KLayout and stores the result in `6_drc_count.rpt`

### CODE REVIEW

**Comment by rovinski** [comment]
Did you make this or was it copied from somewhere? You need to include the license if it originated elsewhere. Also be sure to fix the DCO.

**Comment by yathAg** [comment]
This is based available at https://github.com/laurentc2/ASAP7_for_KLayout. I modified it to work with ORFS makefile and tested it on the UART and gcd designs. 

I can append the Licence at the start of the .lydrc file?

File: flow/platforms/asap7/drc/asap7.lydrc [hunk 1]
```diff
@@ -0,0 +1,558 @@
...
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+###############################################################################
```
Comment (rovinski) [inline flow/platforms/asap7/drc/asap7.lydrc]: I don't think this is valid XML. You need need to enclose it in `<!--` and `-->` for an XML comment.

Comment (yathAg) [reply flow/platforms/asap7/drc/asap7.lydrc]: I just checked the Lissence cannot be added to the start of the .lydrc file. Is it okay if I include the Licence file in the DRC folder?

Comment (rovinski) [reply flow/platforms/asap7/drc/asap7.lydrc]: Does enclosing it in a comment not work? If not, you could move it down to line 47 where the code starts.

**Comment by vijayank88** [comment]
@yathAg 
Can you rebase your repo and push again to check the PR status?
Is LVS check resolved?

**Comment by yathAg** [comment]
Sorry for the late reply @vijayank88 , I guess I have made a mistake in the rebase procedure. Should I close this pull request and open another one?

**Comment by rovinski** [comment]
@yathAg you can always force push your changes to the fork. To me, this is the preferred methodology rather than closing a PR and opening a new one.

**Comment by yathAg** [comment]
> @yathAg you can always force push your changes to the fork. To me, this is the preferred methodology rather than closing a PR and opening a new one.

Thank you for the input, it was quite an exercise figuring out this aspect of git... But seems like I have got it now!

**Comment by rovinski** [comment]
It looks like your latest push may have only updated one of your files. You may want to check that all of the files are updated and then push again.

**Comment by maliberty** [comment]
@rovinski are all of your concerns addressed?

**Comment by rovinski** [comment]
I think so, other than the failing CI for some reason. KLayout DRC isn't part of the CI so there isn't a way to test that this file actually works, but from visual inspection it looks ok.

**Review by yathAg — COMMENTED** [review]
Rechecked and reviewed files.

**Comment by vijayank88** [comment]
@maliberty 
Can you please re-run CI for this PR?

Still #854 under discussion related to LEF file issue.

**Comment by maliberty** [comment]
@vijayank88 it looks like the CI is running already

**Comment by QuantamHD** [comment]
Gentle pin @maliberty Should we rerun the CI?

**Comment by maliberty** [comment]
Thanks for the ping @QuantamHD .  The CI passed but the GH action didn't get updated.

**Pull request merged**